src/course-home/progress-tab/grades/detailed-grades/DetailedGrades.jsx

```diff
@@ -63,14 +67,18 @@ function DetailedGrades({ intl }) {
       {!hasSectionScores && (
         <p className="small">{intl.formatMessage(messages.detailedGradesEmpty)}</p>
       )}
-      <p className="x-small m-0">
-        <FormattedMessage
-          id="progress.ungradedAlert"
-          defaultMessage="For progress on ungraded aspects of the course, view your {outlineLink}."
-          description="Text that precede link that redirect to course outline page"
-          values={{ outlineLink }}
-        />
-      </p>
+      {
+        overviewTabUrl && (
```

Comment:
```suggestion
      {overviewTabUrl && (
```

Let's stick to the indentation pattern set above.